src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -288,6 +291,29 @@ def get_composite_package_identifier(self, package, package_version):
 
     def install_updates_fail_safe(self, excluded_packages):
         return
+
+    def accept_eula_for_patches(self):
```

Comment:
**Important - High level notes**
Separation of concerns not met -- Separate the configuration read from the use of that configuration.

Specifically:
- ConfigurationManager will get a new property that will:
 (a) today: be read from disk (leave inline comment that this is temporary and will be deprecated no later than x date)
 (b) tomorrow: come from full VM model configuration (like other properties there)
The source of this configuration is fully abstracted away - package managers don't need to know.

- All that happens in the Apt package manager is modification of the behavior based on the configuration at initialization. Since this PR only does it for Apt, scope it as such (including title) - doesn't need to be added to the base PackageManager.

- In the Apt class constructor (code simplicity note - 1 new line, and a couple of changed lines; no broader changes) -
(a) optional_accept_eula_cmd_suffix = " ACCEPT_EULA=Y" if self.execution_config.accept_package_eula else ""

(b) each existing line can be changed to this:
self.<whatever command> = <existing command string> + optional_accept_eula_cmd_suffix

Reply:
The loaded configuration (irrespective of load source) can be reused elsewhere (in Zypper for example where there's an open conversation with IDC about this), but we're choosing not to right now. There shouldn't be any additional code, just moving out the read from Apt to ConfigMan - if there is non-trivial increase in code, I may be missing something, let's discuss.

Reply:
Updated the code as per my understanding, please review

Reply:
I was looking at the latest code and trying to map it back to what I had written here --- I was referring to 'ExecutionConfig' here and not 'ConfigurationFactory' here, when I quickly wrote 'ConfigurationManager':

Reply:
done